### DISCUSSION TITLE
Closing of old issues

### DISCUSSION THREAD

**Discussion by lordfolken** [XCSoar/XCSoar#1136] [General]
Hello, 

As you all know there is a high amount of issues that are tagged as enhancement. They would be nice to have etc. but no one has volunteered to implement the idea. 

While ideas are great they are less than 1% of the work. Without having the man power behind it it doesn't make sense to keep these issues around. 

I would propose to configure an issue management bot. This would close issues that have:
 * no one is assigned
 * ticket isn't triaged by project
 * is not part of a milestone
 * has reached a certain age without activity for example 45 days
 * isn't labeld as bug

The bot would mark the ticket as stale around 40 days and close at 45 days. 

What do you think?

**Comment by lordfolken** [comment]
There seems to be a github action: https://github.com/actions/stale that closes not just issues, but also PRs that are inactive.

**Comment by hjr** [comment]
Sounds reasonable to me.

**Comment by MaxKellermann** [comment]
What's the point of that? Issues should be closed if it is decided that they will not be worked on, but without a decision, they should not be closed.

**Reply by lordfolken** [reply]
Leaving an idea in the bug tracker for years and years is exactly the decision that its not being worked on. Because it in fact it isn't. 

The  point  is to shift the burden to the creator of the issue. 


We should steer any would-be contributors to the points that we want to focus on. Leaving 100s of tickets open risks people focusing on the wrong issues, and will lead to random contributions that cater to a single users's use case instead of things that matter to the core.



Ideas which don't gain traction should be discarded. Until either a convincing case by the initiator is made or a volunteer that implements has been found. 







If someone has a strong conviction of an idea, they will implement it, regardless of there being an issue or not. 

Issues which I think that matter i triaged, and put them in the [github projects](https://github.com/XCSoar/XCSoar/projects?type=classic)
(sadly way too hard to find...)
At least for me that is a decision that the idea/issue has merit and should be focused on.
